src/iface-linux.c

```diff
@@ -43,24 +43,21 @@
 #endif
 
 static int netlink_list_items(CattaNetlink *nl, uint16_t type, unsigned *ret_seq) {
-    struct nlmsghdr *n;
+    struct nlmsghdr n = {0, 0, 0, 0, 0};
     struct rtgenmsg *gen;
-    uint8_t req[1024];
```

Comment:
what if they want a 1024 size byte buffer to hold some temporary data. Since you have deleted it, we could run into some problems. Please revisit this...perhaps explore the use of aligned directive...read more here: https://stackoverflow.com/questions/14332633/attribute-packed-v-s-gcc-attribute-alignedx